client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -197,7 +218,14 @@ const LowerJaw = ({
           >
             <Reset />
           </button>
-
+          {challengeIsCompleted &&
+            endOfProject.current &&
+            completedPercent === isBlockCompleted && (
```

Comment:
Can you turn this if statements into a variable for readiblity, something like 

```ts
const showShareButton = challengeIsCompleted &&
            endOfProject.current &&
            completedPercent === isBlockCompleted

     {showShareButton ** (
```

Reply:
updated thanks for suggesting